src/components/Checkout/CheckoutReceipt.vue

```diff
@@ -198,65 +208,68 @@
 					>
 						<span class="total__header tw-text-h3">Kiva Card:</span>
-						<span class="total__amount tw-text-h3">
+						<span class="total__amount tw-text-h3" data-testid="kcard-payment-amount">
 							${{ receipt.totals.redemptionCodeAppliedTotal }}
 						</span>
 					</li>
 					<li
-						data-testid="free-trial"
+						data-testid="payment-type-free-trial"
 						class="total"
 						v-if="receipt.totals.freeTrialAppliedTotal > 0"
 					>
 						<span class="total__header tw-text-h3">Free Trial:</span>
-						<span class="total__amount tw-text-h3">Free!</span>
+						<span class="total__amount tw-text-h3" data-testid="free-trial-payment-amount">Free!</span>
 					</li>
 					<li
-						data-testid="free-credit"
+						data-testid="payment-type-free-credit"
 						class="total"
 						v-if="receipt.totals.bonusAppliedTotal > 0"
 					>
 						<span class="total__header tw-text-h3">Free credit:</span>
-						<span class="total__amount tw-text-h3">{{ receipt.totals.bonusAppliedTotal }}</span>
+						<span class="total__amount tw-text-h3"
+							data-testid="free-credit-payment-amount"
+						>{{ receipt.totals.bonusAppliedTotal }}</span>
 					</li>
 					<li
-						data-testid="kiva-credit-added"
+						data-testid="payment-type-kiva-credit-added"
 						class="total"
 						v-if="receipt.totals.depositTotals.kivaCreditAdded > 0"
 					>
 						<span class="total__header tw-text-h3">Kiva credit added:</span>
-						<span class="total__amount tw-text-h3">
+						<span class="total__amount tw-text-h3" data-testid="kiva-credit-added-payment-amount">
 							${{ receipt.totals.depositTotals.kivaCreditAdded }}
 						</span>
 					</li>
 					<li
-						data-testid="kiva-credit-used"
+						data-testid="payment-kiva-credit-used"
 						class="total"
 						v-if="receipt.totals.depositTotals.kivaCreditUsed > 0"
 					>
 						<span class="total__header tw-text-h3">Kiva credit:</span>
-						<span class="total__amount tw-text-h3">
+						<span class="total__amount tw-text-h3" data-testid="kiva-credit-use-payment-amount">
 							${{ receipt.totals.depositTotals.kivaCreditUsed }}
 						</span>
 					</li>
 					<li
-						data-testid="amount-charged"
+						data-testid="payment-type-amount-charged"
 						class="total"
 						v-if="parseFloat(receipt.totals.depositTotals.depositTotal) > 0"
 					>
 						<span class="total__header tw-text-h3">Amount charged:</span>
-						<span class="total__amount tw-text-h3">
+						<span class="total__amount tw-text-h3" data-testid="amount-charged-payment-amount">
 							${{ receipt.totals.depositTotals.depositTotal }}
 						</span>
 					</li>
 				</ul>
 			</section>
 			<section class="section section--print hide-for-print">
-				<button class="print tw-text-link tw-flex tw-items-center tw-gap-1" @click="printReceipt">
+				<button data-testid="print-receipt"
+					class="print tw-text-link tw-flex tw-items-center tw-gap-1" @click="printReceipt">
```

Comment:
The lint warning here is b/c the `>` should be on the next line. Additionally, there is a 3 item limit for a single line of attributes along with a line-length limit which I'm sure you've hit.

Once we move to multiple lines, we usually also move to one attribute per line to clean it up like I've shown below.

```
<button
    data-testid="print-receipt"
    class="print tw-text-link tw-flex tw-items-center tw-gap-1"
    @click="printReceipt"
>
```

Reply:
Okay, I made the change to fix the link warning.